src/Files.App/Filesystem/ListedItem.cs

```diff
@@ -49,6 +49,9 @@ public string ItemTooltipText
 				return $"{"ToolTipDescriptionName".GetLocalizedResource()} {Name}{Environment.NewLine}" +
 					$"{"ToolTipDescriptionType".GetLocalizedResource()} {itemType}{Environment.NewLine}" +
 					$"{"ToolTipDescriptionDate".GetLocalizedResource()} {ItemDateModified}" +
+					(!String.IsNullOrWhiteSpace(FileSize)
+						? $"{Environment.NewLine}{"ToolTipDescriptionSize".GetLocalizedResource()} {FileSize}"
+						: string.Empty) +
 					(SyncStatusUI.LoadSyncStatus
 						? $"{Environment.NewLine}{"syncStatusColumn/Header".GetLocalizedResource()}: {syncStatusUI.SyncStatusString}"
 						: string.Empty);
```

Comment:
This should do the trick and still working. Like so, no need to come back for a refactoring we'll never make 'cause it will be forgotten.

```suggestion
				var tooltipBuilder = new StringBuilder();
				tooltipBuilder.AppendLine($"{"ToolTipDescriptionName".GetLocalizedResource()} {Name}");
				tooltipBuilder.AppendLine($"{"ToolTipDescriptionType".GetLocalizedResource()} {itemType}");
				tooltipBuilder.Append($"{"ToolTipDescriptionDate".GetLocalizedResource()} {ItemDateModified}");
				if(!string.IsNullOrWhiteSpace(FileSize))
					tooltipBuilder.Append($"{Environment.NewLine}{"ToolTipDescriptionSize".GetLocalizedResource()} {FileSize}");
				if(SyncStatusUI.LoadSyncStatus)
					tooltipBuilder.Append($"{Environment.NewLine}{"syncStatusColumn/Header".GetLocalizedResource()}: {syncStatusUI.SyncStatusString}");

				return tooltipBuilder.ToString();
```